pkg/binding/asker/survey_asker.go

```diff
@@ -40,14 +40,19 @@ func (s *Survey) AskServiceBindingName(defaultName string) (string, error) {
 }
 
 func (o *Survey) AskBindAsFiles() (bool, error) {
-	question := &survey.Confirm{
-		Message: "Bind as files?",
-		Default: true,
+	question := &survey.Select{
+		Message: "How do you want to bind the service?",
+		Options: []string{"Bind as Files", "Bind as Environment Variables"},
 	}
-	var answer bool
+	var answer string
 	err := survey.AskOne(question, &answer)
 	if err != nil {
-		return false, err
+		return true, err
 	}
-	return answer, nil
+
+	var bindAsFiles bool
+	if answer == "Bind as Files" {
```

Comment:
IMO, you should avoid testing the literal value directly. You could pass an int as AskOne to get the index of the result, and test on the index.

Reply:
Right, I keep forgetting about using the index.

Reply:
I do not want to return the int value here, I want to evaluate the `bindAsFiles`'s bool here. So, even if I use int to store the value of user's answer, I would rely on the options to maintain. Something like below - 
```go
func (o *Survey) AskBindAsFiles() (bool, error) {
	options := []string{"Bind as Files", "Bind as Environment Variables"}
	question := &survey.Select{
		Message: "How do you want to bind the service?",
		Options: options,
	}
	var answer int
	err := survey.AskOne(question, &answer)
	if err != nil {
		return true, err
	}

	var bindAsFiles bool
	if answer == 0 {
		bindAsFiles = true
	}
	return bindAsFiles, nil
}
```
This doesn't seem correct to me.
Is this what you were suggesting?

Reply:
Const sounds like a better idea, will do that.